lib/taskana-core/src/main/java/pro/taskana/task/api/models/TaskSummary.java

```diff
@@ -38,133 +39,176 @@
   /**
    * Gets the time when the task was created.
    *
-   * @return the created Instant
+   * @return the created {@link Instant}
    */
   Instant getCreated();
 
   /**
-   * Gets the time when the task was claimed.
+   * Gets the time when the Task was claimed.
    *
-   * @return the claimed Instant
+   * @return the claimed {@link Instant}
    */
   Instant getClaimed();
 
   /**
-   * Gets the time when the task was completed.
+   * Gets the time when the Task was completed.
    *
-   * @return the completed Instant
+   * @return the completed {@link Instant}
    */
   Instant getCompleted();
 
   /**
-   * Gets the time when the task was last modified.
+   * Gets the time when the Task was last modified.
    *
-   * @return the last modified Instant
+   * @return the last modified {@link Instant}
    */
   Instant getModified();
 
   /**
-   * Gets the time when the task is planned to be executed.
+   * Gets the time when the Task is planned to be executed.
    *
-   * @return the planned Instant
+   * @return the planned {@link Instant}
    */
   Instant getPlanned();
 
   /**
-   * Gets the time when when the surrounding process was started.
+   * Gets the time when the surrounding process was started.
    *
-   * @return the received Instant
+   * @return the received {@link Instant}
```

Comment:
I don't think we link Instants, since it's not an entity of us, but you can leave this for now :)